src/handlers/query.ts

```diff
@@ -249,42 +245,77 @@ async function executeSingleQuery(
       })
     )
   );
-  if (query.aggregates) {
-    const aggregateCount = await executeAggregateQuery(query, config, table);
-    const aggregates = {
-      aggregate_count: {
-        [table]: aggregateCount,
+  if (query.aggregates && query.where ) {
```

Comment:
Didn't notice anything in which aggregates **required** a `where`. Any reason this is included?

Reply:
that should just be `query.aggregates` will change that